examples/react_webpack/BUILD.bazel

```diff
@@ -13,22 +13,14 @@ sass(
     data = ["styles.scss"],
 )
 
-tsc(
-    name = "compile",
-    outs = ["index.js"],
-    args = [
-        "$(execpath index.tsx)",
-        "$(execpath types.d.ts)",
-        "--outDir",
-        "$(RULEDIR)",
-        "--lib",
-        "es2015,dom",
-        "--jsx",
-        "react",
-    ],
-    data = [
+ts_project(
+    name = "tsconfig",
+    srcs = [
         "index.tsx",
         "types.d.ts",
+    ],
+    declaration = False,
```

Comment:
Can you a comment describing why declaration is set to false?

Reply:
Looks like it's a demo showing a 1:1 with the old `tsc`  which didn't do declarations

Reply:
added a comment, yes this is exercising the codepath where tsc wasn't configured with `declaration` or `composite`